src/playercards/customizable/UpgradeSheetLibrary.ttslua

```diff
@@ -152,22 +166,30 @@ function createRowTextField(rowIndex)
     rowInputIndex[rowIndex] = #previousInputs
   end
   local funcName = "textbox" .. rowIndex
-  local func = function(_, _, val, sel) clickTextbox(rowIndex, val, sel) end
+  local func = function(player, value) clickTextbox(rowIndex, value) end
   self.setVar(funcName, func)
 
-  self.createInput({
-    input_function = funcName,
-    function_owner = self,
-    label          = "Click to type",
-    alignment      = 2,
-    position       = textField.position,
-    scale          = { 0.1, 0.1, 0.1 },
-    width          = textField.width * 10,
-    height         = inputFontsize * 10 + 75,
-    font_size      = inputFontsize * 10.5,
-    color          = "White",
-    value          = ""
-  })
+  local posVector = Vector(textField.position)
+  local actualPosition = tostring(posVector.x * -100) .. " " .. tostring(posVector.z * 100) .. " " .. tostring(-22)
```

Comment:
I think it would be good if this conversion was a separate function for readability (maybe even combined with the checkbox one?)